NEFSC/READ-SSB-CHAJI-Effort-Displacement---Scallop
# Package tidy

Tidy up loading in packages.  

To test this out, I:
1.  loaded all the packages in the 07a7433 order 
```
PKG <- c("leaflet", "tidyverse", "sf",  "RODM", "dbplyr","raster","rgdal","readxl", "data.table")
```
one by one and looked to see if there were any commands masked off.   
2.  Inspected the code to see if any of those commands were used.  The only one is *filter*, and it's clear from the context that we intend to use the dplyr::filter() and not stats::filter().

So that's it.


tidyverse loads ggplot2, dplyr, and readr.
raster loads sp.

Code is tested functional (with the exception of the "reading_spatial_join" chunk).
